desktop/app.js

```diff
@@ -3,9 +3,14 @@ var Menu = require( 'menu' );
 var BrowserWindow = require( 'browser-window' );	// Module to create native browser window.
 var path = require('path')
 
-module.exports = function main( url ) {
+require( 'module' ).globalPaths.push( path.resolve( path.join( __dirname ) ) );
+
+module.exports = function main() {
+	var url = 'file://' + path.join( __dirname, '..', 'dist', 'index.html' );
```

Comment:
yeah - I wouldn't ever recommend that we detect and replace the slashes :wink:

this is fine. I think ideally we'd use a URL-builder, but if IE/Chrome/Firefox in Windows handle the wrong slash it's okay to ship.